python/src/trezorlib/debuglink.py

```diff
@@ -239,11 +275,12 @@ def button_request(self, br):
             raise AssertionError("input flow ended prematurely")
         else:
             try:
+                assert isinstance(self.input_flow, Generator)
```

Comment:
this does not look like something that works at runtime.
please try running the test suite, I have a suspicion that this will fail.

you'll probably need this: https://stackoverflow.com/a/6416585/222189